studio/schemas/documents/leaderCard.js

```diff
@@ -18,5 +18,10 @@ export default {
             title: 'Image',
             type: 'mainImage',
         },
+        {
+            name: 'linkedinURL',
+            title: 'LinkedIn URL',
+            type: 'url',
+        },
```

Comment:
What is this for? Are they displaying only their linkedin url?

Reply:
@asasmith I thought we wanted to link the card to their linkedin. let me confirm with emily

Reply:
But to answer your question, yes they want their LinkedIn added to the leader card. I am confirming the logic with emily.